src/plugins/legacy/itkFilters/CMakeLists.txt

```diff
@@ -73,8 +73,8 @@ add_library(${TARGET_NAME} SHARED
 target_link_libraries(${TARGET_NAME}
   ${QT_LIBRARIES}
   dtkCore
-  dtkLog
-  ITKCommon
+  dtkLog  
+  ${ITK_LIBRARIES}
```

Comment:
Ugly, can't we use only the right libs ?

Reply:
Normally: When you specify a list of component on find_package the ${XXX_LIBRARIES} contain only needed  libraries.
I'm agree with you for the moment I still to prefer manually indicate all libraries and I wait medInria4.0 to investigate evaluate the interest to use  ${XXX_LIBRARIES}.
But for some linkerrors I couldn't determine which lib is needed. So I exceptionally used  ${XXX_LIBRARIES} and I don't generalize this process.

For  find_package without component list,  ${XXX_LIBRARIES} is still totally forbidden for me.